sklearn/feature_selection/tests/test_feature_select.py

```diff
@@ -667,3 +667,16 @@ def test_mutual_info_regression():
     gtruth = np.zeros(10)
     gtruth[:2] = 1
     assert_array_equal(support, gtruth)
+
+
+def test_generic_univariate_selection_empty_selection():
```

Comment:
I don't think actually need this test. The `check_transformer_preserve_dtypes` common test, which should be activated with the `preserves_dtype` estimator tag should check this already.